src/main/java/org/agbrothers/englishbot/constant/MessageLabel.java

```diff
@@ -8,5 +8,8 @@ private MessageLabel() {
     public static final String NEXT_LESSON = ",\nдля продовження навчання оберіть урок:";
     public static final String CORRECT_ANSWER = "Ви відповіли правильно! \n";
     public static final String INCORRECT_ANSWER = "Відповідь не вірна! Правильна відповідь - ";
-
+    public static final String NEWLINE = "\n";
+    public static final String PROBEL = " ";
+    public static final String FROM = "із";
```

Comment:
I suppose there are no so many usages of this string in code, hence it does'nt need to be constant